src/ng/templateRequest.js

```diff
@@ -36,7 +36,8 @@ function $TemplateRequestProvider() {
 
       var httpOptions = {
         cache: $templateCache,
-        transformResponse: transformResponse
+        transformResponse: transformResponse,
+        headers: { 'Accept': 'text/html, text/ng-template' }
```

Comment:
Agreed. The proposal here sounds like a good default value.

BTW, this needs tests.